docs/04-java-client/01-client-overview.md

```diff
@@ -6,36 +6,73 @@ permalink: /docs/java-client/client-overview
 
-### com.uber.cadence.client
-APIs for external application code to interact with Cadence workflows: start workflows, send signals or query workflows.
+### [com.uber.cadence.client](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/client/package-summary.html)
+Client to the Cadence service used to start and query workflows by external processes
 
-### com.uber.cadence.workflow
+### [com.uber.cadence.workflow](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/workflow/Workflow.html)
 APIs to implement workflows.
 
-### com.uber.cadence.worker
+### [com.uber.cadence.worker](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/worker/package-summary.html)
 APIs to configure and start workers.
 
-### com.uber.cadence.testing
+### [com.uber.cadence.testing](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/testing/package-summary.html)
 APIs to write unit tests for workflows.
 
-## [Samples](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples)
-### com.uber.cadence.samples.hello
+## Cadence Java Samples
+### [com.uber.cadence.samples.hello](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples/hello)
 Samples of how to use the basic feature: activity, local activity, ChildWorkflow, Query, etc.
 This is the most important package you need to start with.
-### com.uber.cadence.samples.bookingsaga
+### [com.uber.cadence.samples.bookingsaga](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples/bookingsaga)
 An end-to-end example to write workflow using SAGA APIs.
-### com.uber.cadence.samples.fileprocessing
+### [com.uber.cadence.samples.fileprocessing](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples/fileprocessing)
 An end-to-end example to write workflows to download a file, zips it, and uploads it to a destination.
 
  An important requirement for such a workflow is that while a first activity can run
 on any host, the second and third must run on the same host as the first one. This is achieved
  through use of a host specific task list. The first activity returns the name of the host
   specific task list and all other activities are dispatched using the stub that is configured with
  it. This assumes that FileProcessingWorker has a worker running on the same task list.
+
+
+## Differences between Java and Golang Clients
+
+While many features are already supported in the Cadence Golang client, some features are still missing in the Java client. The Cadence development team aims for feature parity between the two clients, with new features typically being implemented in Golang first.
+
+### Feature Comparison
+
+| Feature                                                   | Go Client      | Java Client   | Priority |
```

Comment:
It might make sense to create a github issue for missing functionality in the cadence-java-client repository and link it here. That would provide additional context for readers and also provide a starting point for new contributors. Since there's an existing implementation to look at it's much more straightforward to go about implementing.

I think this list is also heavily biased towards things that we don't have since we were tracking this for the purpose of improving the Java client. Generally it has all the core features that you'd expect in a Cadence client. Workers, Workflows, Activities, Local Activities, Locally Dispatched Activities, Signals, SignalWithStart, Querying, Cancellation, etc.

Reply:
Thanks! added a link to the issues where applicable!